bin/nodist

```diff
@@ -1,10 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
-if [ "$1" = "env" -o "$1" = "use" ]; then
-    \nodist + $2
-    if [ $? -eq 0 ]; then
-        export NODIST_VERSION=$2
-    fi
-else
-    \nodist $*
-fi
\ No newline at end of file
+DIR=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
+"$DIR/../node.exe" "$DIR/../cli" "$@"
```

Comment:
Mh ok. This is a different issue than what was reported, though. It looks like cygwin isn't converting the path at all and node just tries to sanitize it, which fails of course, because node doesn't know it's being run in cygwin. :/